csrankings-d.csv

```diff
@@ -59,6 +59,7 @@ Dale F. Reed,University of Illinois at Chicago,https://sites.google.com/site/dal
 Dale R. Thompson,University of Arkansas,https://www.netgeekdr.com,NOSCHOLARPAGE
 Dale Reed,University of Illinois at Chicago,https://sites.google.com/site/dalereed,NOSCHOLARPAGE
 Dale Schuurmans,University of Alberta,https://webdocs.cs.ualberta.ca/~dale,xaQuPloAAAAJ
+DAljit Singh Dhillon,Clemson University,https://djsingh.people.clemson.edu/,3GWWmEsAAAAJ

```

Comment:
Mis-capitalized and needs another entry to account for DBLP alias. https://dblp.org/pid/14/7591.html

Reply:
Updated.